architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,86 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization determines whether an identity is allowed to perform any action.
+Note that authorization is a separate step from
+link:authentication.html[authentication], which is more about determining the
+identity of who is taking the action.
+
+Authorization is managed using:
+
+[horizontal]
+Rules:: Describe sets of permitted verbs on a set of resource types.
+Roles:: Collection of rules.
+Bindings:: Associate users and/or groups with a role.
+
+Users and groups may be bound to multiple roles at the same time.
+
+Several factors are combined to make the decision when authorization is
+evaluated:
+
+[horizontal]
+Identity:: Both the username and list of groups the user belongs to.
+Action:: The action being performed. In most cases, this consists of:
+Project::: The project being accessed.
+Verb::: Can be `get`, `list`, `create`, `update`, or others.
+Resource::: The API endpoint being accessed. This is distinct from the resource
+being touched, such as pods, `*deploymentconfigs*`, or builds.
+Bindings:: The full list of bindings.
+
```

Comment:
I'm not so sure about these lists, and we should probably try to avoid such nested lists. Wouldn't a table work here?